subsys/gazell/Kconfig

```diff
@@ -34,6 +34,28 @@ config GAZELL_PAIRING_HOST
 	bool "Host"
 	depends on ENTROPY_GENERATOR
 
+choice GAZELL_PAIRING_CONFIG
```

Comment:
Please move the GAZELL_PAIRING_CONFIG choice outside of the GAZELL_PAIRING_ROLE choice.

Reply:
Move misplace config inside `GAZELL_PAIRING_ROLE` block.